core/src/banking_stage/tpu_to_pack.rs

```diff
@@ -199,25 +228,33 @@ mod tests {
     fn test_copy_packet_and_populate_message() {
         let packet_bytes = vec![1, 2, 3, 4, 5];
         let src_ip = Ipv4Addr::new(192, 168, 1, 1);
-        let packet_meta = solana_packet::Meta {
-            size: packet_bytes.len(),
-            addr: IpAddr::V4(src_ip),
-            port: 1,
-            flags: PacketFlags::all(),
-        };
+        let mut packet_meta = solana_packet::Meta::default();
```

Comment:
This is because of the encapsulation of the remote_pubkey which is declared private. We probably should have a constructor to ease it.